### DISCUSSION TITLE
How secure code search?

### DISCUSSION THREAD

**Discussion by warengonzaga** [community/community#8583] [Other Feature Feedback, Questions, & Ideas] [answered]
Like other search engines, they can be used for bad activities. For example, Google has google dorks. So are we expecting a GitHub Code Search Dorks? 🤔 How secure code search is? I'm curious...

**Comment by tclem** [comment, marked as answer]
Code search is built to the same security standards as the rest of GitHub with the exact same access policies for private repositories. It's very secure.

We take security and private repository access seriously. You can read our high level [security page](https://github.com/security) for more details. I will also note that the code search preview is part of our [Security Bug Bounty](https://bounty.github.com/) program.
